drivers/sdcard_spi/Kconfig

```diff
@@ -15,3 +15,8 @@ config MODULE_SDCARD_SPI
     select MODULE_PERIPH_SPI_RECONFIGURE if HAS_PERIPH_SPI_RECONFIGURE
     select MODULE_CHECKSUM
     select ZTIMER_USEC
```

Comment:
I think we also need to `select MODULE_ZTIMER`

Reply:
I can do that, just keep in mind, currently the `ZTIMER_USEC` selects `MODULE_ZTIMER`, some devices bring in both, some do not.  I will add it now, if in the future there is a remodelling of the kconfig ZTIMER stuff it will likely require the `MODULE_ZTIMER` entrypoint anyways.